extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1841,10 +1847,14 @@ export class AmpA4A extends AMP.BaseElement {
    * @return {string} full url to safeframe implementation.
    */
   getSafeframePath() {
-    return (
-      'https://tpc.googlesyndication.com/safeframe/' +
-      `${this.safeframeVersion}/html/container.html`
-    );
+    const subdomain = isExperimentOn(
```

Comment:
Couple of issues:

1. You need to divert the experiment in both adsense & doubleclick impls if you want it to run in both... up to you if you want to use the same experiment ids or not
2. Given you're diverting in the impls, you don't want/need to modify amp-a4a.js.  Instead you should just override getSafeframePath() in each of those impls.
3.  You don't want to use isExperimentOn as that only indicates if AMP diverted into the "global" experiment when you want to know if you're also not in the control branch.  Code example:

/** @override */
getSafeframePath() {
   if (!this.experimentIds.includes(
      RANDOM_SUBDOMAIN_BRANCHES.RANDOM_SUBDOMAIN_EXPERIMENT)) return super.getSafeframePath();
   // call and construct safeframe path here....